src/TimeManager.jl

```diff
@@ -1,9 +1,9 @@
-#=
+"""
     TimeManager
 
 This module facilitates calendar functions and temporal interpolations
 of data.
-=#
+"""
```

Comment:
This is still broken due to the newline between the triple quotes and the module

Reply:
Oh, thank you. I was checking the local build and did not realize. Thanks